plugins/outputs/kafka/kafka_test.go

```diff
@@ -25,10 +25,11 @@ func TestConnectAndWrite(t *testing.T) {
 	brokers := []string{testutil.GetLocalHost() + ":9092"}
```

Comment:
Add a case with non-default `MaxBatchRetry` @BenMatase?

also not sure if we can simulate the `sarama` failures in these tests - donno if they are already mocking out the `sarama` library APIs somehow.